src/components/shared/ArrowPanel.css

```diff
@@ -103,6 +103,7 @@
 .arrowPanelSubTitle {
   padding: 5px 0;
   border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+  font-weight: 700;
```

Comment:
had to add this. It was implicit before (because of the `<h2>` tag, but now we gave it to `<summary>`)